iam_builder/templates.py

```diff
@@ -248,6 +260,28 @@ def get_s3_list_bucket_policy(list_of_buckets):
     }
     return policy
 
+def get_role_secrets(role):
+    if len(role)>1:
```

Comment:
I would have role as a string and not a list.

So in the yaml file expect something like `secrets_role_name: role_name` and if it exists as a string then add the parameter statements in - including the decrypt kms one. Then you wouldn't have to check for length.

Reply:
I'm wondering if we should just have a `role_name` param and then set glue and secrets to boolean instead?

Otherwise people are writing in their role twice for glue and secrets (if they would even need both). 

Do we think that there would be an instance where your glue_name != secrets_role_name?